{{cookiecutter.repo_name}}/{{cookiecutter.app_name}}/__init__.py

```diff
@@ -1 +1,2 @@
 __version__ = '{{ cookiecutter.version }}'
+default_app_config = '{{ cookiecutter.app_name }}.apps.{{ cookiecutter.app_config_name }}'
```

Comment:
Please remove this change. See https://docs.djangoproject.com/en/1.10/ref/applications/#configuring-applications:

> **New applications should avoid default_app_config**. Instead they should require the dotted path to the appropriate AppConfig subclass to be configured explicitly in INSTALLED_APPS.